storage/local/storage.go

```diff
@@ -212,10 +212,18 @@ func (s *memorySeriesStorage) Start() error {
 		return err
 	}
 	s.persistence = p
+	go s.persistence.run()
+
+	pclose := func() {
```

Comment:
Yeah, the latter is intentional, kind of an "informed risk" the language designers took...